app-website-4/ProductsApi/Auth/TokenProvider.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Microsoft.IdentityModel.Tokens;
+using System;
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using System.Text;
+
+namespace ProductsApi.Auth
+{
+    public class TokenProvider
+    {
+        public string GenerateToken(string[] usernameAndPassword)
```

Comment:
might as well make this take two individual string params:
`username` and `password`

Reply:
Yeah, I thought about doing this anyways, I agree.